storybook/src/components/Tabs/Tabs.stories.tsx

```diff
@@ -40,41 +40,50 @@ const tabMeta = {
 type Story = StoryObj<typeof meta>
 type TabStory = StoryObj<typeof tabMeta>
 
+const defaultTabs = [
```

Comment:
You could've put it in the meta constant, like we do everywhere else if there's more then one story using the props.

Reply:
Agreed, I'll adjust that.

Reply:
I was taking a stab at refactoring the example tab template into a configuration object:

```
type TabContent = {
  label: string
  body: ReactNode
}

const tabContent: Array<TabContent> = [
  {
    label: 'Gegevens',
    body: <Paragraph>{exampleParagraph()}</Paragraph>,
  },
(…)
```

Then `defaultTabs` could become a function looping over this data.
I thought it might come closer to how implementers build Tabs.
And we can calculate the max value of `tab` and `activeTab` to show exactly the right amount of controls.

Good idea?